tests/installation/bootloader_zkvm.pm

```diff
@@ -32,12 +33,12 @@ sub set_svirt_domain_elements {
 
         my $ntlm_p = get_var('NTLM_AUTH_INSTALL') ? $ntlm_auth::ntlm_proxy : '';
         my $cmdline = get_var('VIRSH_CMDLINE') . $ntlm_p . " ";
-        if (get_var('AGAMA')) {
-            $cmdline .= "root=live:ftp://" . get_var('REPO_HOST', 'openqa') . '/' . get_var('REPO_999');
+        if (is_agama) {
+            $cmdline .= "root=live:http://" . get_var('OPENQA_HOSTNAME') . "/assets/iso/" . get_required_var('ISO') . " live.password=$testapi::password";
```

Comment:
Although from the job setting, `OPENQA_HOSTNAME` and `OPENQA_URL` are the same, I prefer `OPENQA_URL` here

Reply:
The reason why :) https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20777#discussion_r1881474317